Transfer/Sources/TransferAmountCell.swift

```diff
@@ -109,41 +109,58 @@ final class TransferAmountCell: UITableViewCell {
 
 extension TransferAmountCell: UITextFieldDelegate {
     func textFieldDidEndEditing(_ textField: UITextField) {
-        enteredAmountHandler?(amountTextField.text ?? "")
+        let formattedAmountString = textField.text?.format(with: currencyLabel.text)
```

Comment:
As per discussion with Willem no need of doing this formatting here. So you can remove the formatting here. Once user clicks on next button, api call itself returns formatted amount that will be displayed on the Confirmation page.

Reply:
fixed